public/index.html

```diff
@@ -28,15 +28,30 @@
     </head>
     <body>
         <noscript>You need to enable JavaScript to run this app.</noscript>
-        <div id="root"></div>
-        <!--
-      This HTML file is a template.
-      If you open it directly in the browser, you will see an empty page.
+        <div id="root">
+            <!-- The core Firebase JS SDK is always required and must be listed first -->
+            <script src="https://www.gstatic.com/firebasejs/8.3.0/firebase-app.js"></script>
 
-      You can add webfonts, meta tags, or analytics to this file.
-      The build step will place the bundled scripts into the <body> tag.
+            <!-- TODO: Add SDKs for Firebase products that you want to use
+                https://firebase.google.com/docs/web/setup#available-libraries -->
+            <script src="https://www.gstatic.com/firebasejs/8.3.0/firebase-analytics.js"></script>
 
-      To begin the development, run `npm start` or `yarn start`.
-      To create a production bundle, use `npm run build` or `yarn build`.
-    --></body>
+            <script>
+            // Your web app's Firebase configuration
+            // For Firebase JS SDK v7.20.0 and later, measurementId is optional
+            var firebaseConfig = {
+                apiKey: "%REACT_APP_FIREBASE_API_KEY%",
+                authDomain: "%REACT_APP_FIREBASE_DOMAIN%",
+                projectId: "%REACT_APP_FIREBASE_PROJECT_ID%",
+                storageBucket: "%REACT_APP_FIREBASE_STORAGE_BUCKET%",
+                messagingSenderId: "%REACT_APP_FIREBASE_SENDER_ID%",
+                appId: "%REACT_APP_FIREBASE_APP_ID%",
+                measurementId: "%REACT_APP_FIREBASE_MEASUREMENT_ID%"
+            };
+            // Initialize Firebase
+            firebase.initializeApp(firebaseConfig);
```

Comment:
This html page is for the user side of things (the students hitting the escape room). Should the initialization of firebase happen here, since there would be no auth needed here?

Reply:
We can later move this to a server.js file or something in the future, but those haven't been developed yet. I think it's fine here for now.

Reply:
We want to keep this page as non-expensive to render as possible though. So we shouldn't have any react or Firebase stuff in here. Maybe make a second page for admins only? Like public/admin.html?